apps/monitoring/ptl-intsvc/00/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ resources:
 patches:
 - path: ../../kube-prometheus-stack/ptl-intsvc/00.yaml
 - path: ../../opentelemetry-collector/ptl-intsvc/00.yaml
+- path: ../../version-reporter/helmcharts/environment-patches/ptl-intsvc.yaml
```

Comment:
You dont need the patch here as you have it in the base file: 
[apps/monitoring/ptl-intsvc/base/kustomization.yaml](https://github.com/hmcts/cnp-flux-config/pull/32830/files#diff-031d7d41111cbb2ba528fd43b2e1bd23b8d850336e27fc6c8863de7037046477)

Reply:
i see